learning_observer/learning_observer/routes.py

```diff
@@ -32,6 +33,20 @@
 from learning_observer.log_event import debug_log, startup_state
 
 from learning_observer.utility_handlers import *
+import pmss
+
+
+pmss.register_field(
+    name='source',
+    type='roster_source',
+    description='Source to use for student class rosters. This can be\n'\
+                '`all`: aggregate all available students into a single class\n'\
+                '`test`: use sample course and student files\n'\
+                '`filesystem`: read rosters defined on filesystem\n'\
+                '`google_api`: fetch from Google API\n'\
+                '`canvas`: fetch from Canvas API',
+    required=True
+)
```

Comment:
Why is this here? We already registered this field in `rosters.py`.

Reply:
Removed